physionet-django/console/templates/console/console_navbar.html

```diff
@@ -322,6 +322,16 @@
         </li>
       {% endif %}
 
+      <!-- front pages -->
```

Comment:
I'm not super keen on the way this appears in the console menu, but we can clean this up in a later pull request. I think probably "static page" and "frontpage buttons" could be grouped into a submenu?